### PR TITLE
[PLAYER-5672] Ad timer is not decremented while ad is playing

### PR DESCRIPTION
**Pull request by cacel**
- For Live Assets, the playhead is set by `liveHelper.frozenSeekBackTime`. When the `AdPlaybackScreen` is set, the playhead is always the initialTime and is not changed by playheadUpdate event.
I added a new boolean flag when the AdPod started (true) and AdPod completed (false), if an ad is playing, the playhead will be updated until the adPodCompleted is fired.

### CODE REVIEW

File: sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m [hunk 1]
```diff
@@ -460,15 +462,22 @@ - (void)bridgeAdTappedNotification:(NSNotification *)notification {
 }
 
 - (void)bridgeAdPodStartedNotification:(NSNotification *)notification {
+  _isAdPlaying = YES;
```
Comment (oSulyma) [inline sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m]: Please, use self.isAdPlaying for all cases of using isAdPlaying until it's not setter implementation and not init method

File: sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m [hunk 2]
```diff
@@ -460,15 +462,22 @@ - (void)bridgeAdTappedNotification:(NSNotification *)notification {
...
   [self.ooReactSkinModel sendEventWithName:notification.name body:nil];
 }
 
 - (void)bridgeAdPodCompleteNotification:(NSNotification *)notification {
-  Float64 duration = self.player.duration;
-  Float64 playhead = self.player.playheadTime;
+  _isAdPlaying = NO;
```
Comment (oSulyma) [inline sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m]: The same

**Review by loginov-rocks — COMMENTED** [review]
JS side fixed

**Review by loginov-rocks — APPROVED** [review]

**Review by oSulyma — APPROVED** [review]
iOS-part is ok

File: sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m (hunk 1 above)
Comment (oSulyma) [reply sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m]: Done

File: sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m (hunk 2 above)
Comment (oSulyma) [reply sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m]: Done

**Pull request merged**